src/templates/common/link/index.test.tsx

```diff
@@ -1,38 +0,0 @@
-import { render } from '@testing-library/react'
```

Comment:
All components that use `<Link>` are importing it directly from Next, so this wasn't used.